indico/modules/events/editing/blueprint.py

```diff
@@ -16,53 +16,53 @@
                       virtual_template_folder='events/editing')
 
 # Frontend
+contrib_prefix = '/contributions/<int:contrib_id>/editing/<any(paper,slides,poster):type>'
```

Comment:
Is it better to leave a space between the commas here? ex: <any(paper, slides, poster):type>
If not, I'll change it in my PR also.

Reply:
IMHO it's more readable without the space in this particular case (just like not having spaces around the `=` of kwargs)

Reply:
It seems like other blueprints (for example `indico/modules/rb`) don't have spaces in there as well, so I left them out
I don't know if it's better or not, though